MapboxCoreNavigation/RouteController.swift

```diff
@@ -282,6 +282,17 @@ open class RouteController: NSObject, Router {
         let isCloseToCurrentStep = location.isWithin(radius, of: routeProgress.currentLegProgress.currentStep)
         return isCloseToCurrentStep
     }
+    
+    /**
+     Advances the leg index.
+     
+     This is a convienence method provided to advance the leg index of any given router without having to worry about the internal data structure of the router.
+     */
+    public func advanceLegIndex(location: CLLocation) {
```

Comment:
Is this method public solely in order to make it available to MapboxNavigation, or is it intended to be used by application code? If application code can use it, let’s make sure to expose it to Objective-C.